src/drive/ducks/services/FuzzyPathSearch.js

```diff
@@ -84,12 +85,13 @@ class FuzzyPathSearch {
             wordsFromPreviousQuery.push(currentQuerySegment)
           }
 
+          wasInPreviousQuery = true
           continue
         }
       }
 
       // this segment wasn't included in any previous query segment so it's a new word and we prioritize it
-      priorizedWords.push(currentQuerySegment)
+      if (!wasInPreviousQuery) priorizedWords.push(currentQuerySegment)
```

Comment:
Before this fix, if we ran the exact same query twice in a row, we would accidentally end up duplicating the query words, ie. `['admin']` would turn into `['admin', 'admin']`

Reply:
a Set instead of an Array could let us add only once the same item: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
I think the use of this variable is very ugly, but the double for loop and the double if statement are too. So that is not a prioritize.